src/diff-snapshot.js

```diff
@@ -213,7 +213,7 @@ function diffImageToSnapshot(options) {
         diffOutputPath,
         diffRatio,
         diffPixelCount,
```

Comment:
Can we add an integration test for this?

Reply:
An existing spec was actually doing this incorrectly, so I fixed it.